packages/live-share-media/src/MediaPlayerSynchronizer.ts

```diff
@@ -19,6 +19,10 @@ export interface IMediaPlayerSynchronizerEvent extends IEvent {
      * Event details.
      */
     details: ExtendedMediaSessionActionDetails;
+    /**
+     * Player error
+     */
+    playerError?: DOMException;
```

Comment:
>playerError [](http://example.com/codeflow?start=4&length=11)

I would just call this "error" and then add a try { } catch() {} around the entire action handler block below